.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
```

Comment:
Let's just go straight to Github Actions :).

Let me know if you'd like help making the config!